### ISSUE TITLE
`master` build is broken - but builds locally?

### ISSUE THREAD

**Issue by colemickens** [nix-community/nixpkgs-wayland#364]
This is bizarre:

```
trying https://github.com/Osleg/gebaar-libinput-fork/archive/6ce14eb64a9f4bb8201bae26052521ae5e64f553.tar.gz
100 24180    0 24180    0     0   112k      0 --:--:-- --:--:-- --:--:--  112k
unpacking source archive /build/6ce14eb64a9f4bb8201bae26052521ae5e64f553.tar.gz
error: hash mismatch in fixed-output derivation '/nix/store/ip8l78731742l2r2zys0jzw9ayipgas7-source.drv':
         specified: sha256-NwF9TOJ6SzJJBa7mrUOCLEhz2xf0it7rVuOzhqvDwt0=
            got:    sha256-9l59FrRUWM6L5dnZm5X/ir/X+ZgzQ2TT5TnNeX1OyIw=
error: 1 dependencies of derivation '/nix/store/54slqk4q963pak3iqyqbar1wra54fkq1-gebaar-libinput-6ce14eb64a9f4bb8201bae26052521ae5e64f553.drv' failed to build
```

gebaar-libinput's source hasn't changed, rev hasn't changed, hash hasn't changed.

Did GitHub change something on their runner, and maybe gebaar-libinput exposes some FS quirk? (I think that's happened before with certain archives and macos targets for example)

**Comment by colemickens** [comment]
Well... if I invalidate the (rev and update to redo the) hash, the hash **does** change:

```
   rev = "6ce14eb64a9f4bb8201bae26052521ae5e64f553";
-  sha256 = "sha256-NwF9TOJ6SzJJBa7mrUOCLEhz2xf0it7rVuOzhqvDwt0=";
+  sha256 = "sha256-9l59FrRUWM6L5dnZm5X/ir/X+ZgzQ2TT5TnNeX1OyIw=";
```

**Comment by colemickens** [comment]
Is this related to nixpkgs's derivation having "fetchSubmodules" for their src?

**Comment by colemickens** [comment]
yup, that's it, and that's why it doesn't build with the updated hash.

**Comment by colemickens** [comment]
fixed in #360
